src/mongocxx/test/pool.cpp

```diff
@@ -96,6 +97,7 @@ TEST_CASE(
     REQUIRE(interposed.crl_file == crl_file);
     REQUIRE(interposed.weak_cert_validation == allow_invalid_certificates);
 }
+#endif
```

Comment:
Can we also test that without SSL support in libmongoc, trying to pass SSL options to a pool or client fails?  Or throws.